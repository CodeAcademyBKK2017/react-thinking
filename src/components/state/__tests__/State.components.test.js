import renderer from 'react-test-renderer';
import React from 'react';
import State from '../State.components';

test('State: snapshot test', () => {
    const snapshot = renderer.create(<State value='1' />);
    expect(snapshot).toMatchSnapshot();
});