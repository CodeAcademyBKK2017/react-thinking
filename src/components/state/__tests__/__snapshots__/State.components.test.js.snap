// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`State: snapshot test 1`] = `
<input
  type="text"
  value="1"
/>
`;
